### ISSUE TITLE
Investigate using PUT (or PATCH) instead of DELETE/POST for ResourceQuota/LimitRanges for Tenant.Init

### ISSUE THREAD

**Issue by aslakknutsen** [openshiftio/openshift.io#2547]
In an attempt to remove cluster scoped role create/delete ResourceQuota/LimitRanges and only rely on update role, investigate how Tenant Init cold use PUT/PATCH instead of DELETE/POST REST APIs.

**Comment by abhgupta** [comment]
Please include rolebindingrestriction resources as well within the scope of this issue.

**Comment by aslakknutsen** [comment]
@abhgupta We do create our own RoleBindingRestrictions tho, not just edit...?

**Comment by abhgupta** [comment]
@aslakknutsen Alright, as part of this review/revisit, I'd like to see what is being created and whether that can be folded into one of the existing rolebindingrestriction resources.

**Comment by aslakknutsen** [comment]
@abhgupta Right. Added. I just remember we never quite got the RBR's to work as 'assumed' when we tried last year. Whatever we got now is kinda whatever we got working last time. So definitely reviewable. :)

**Comment by abhgupta** [comment]
@aslakknutsen rolebindingrestrictions are definitely less of a concern from a security standpoint or the standpoint of accidentally breaking the cluster. But still, something worth reviewing.
